### PR TITLE
[iOS] Apply TCA to AboutDroidKaigi 

### LINKED ISSUE #522: [iOS] Apply TCA to AboutDroidKaigi screen and SettingScreen
## Kind (Required)

- Improvement

 ## Overview (Required)

- Title says all.
- Please apply TCA(The composable architecture) to those screens. (Make `**Feature` file and use `Store` in Screens)

 ## Links	

-

### PR DESCRIPTION
**Pull request by ry-itto**
## Issue
- close #522 

## Overview (Required)
- Regards @Tech-Nayuta 

## Links
- #565 
- #601 

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by ry-itto** [comment]
If following PR is merged before this, I'll apply following PR's content to this PR.
https://github.com/DroidKaigi/conference-app-2021/pull/673

File: ios/Sources/UIApplicationClient/Client.swift [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import Combine
+import UIKit
+
+public protocol UIApplicationClientProtocol {
+    func openSettings() -> AnyPublisher<Bool, Never>
+}
+
+public struct UIApplicationClient: UIApplicationClientProtocol {
+
+    public init() {}
```
Comment (ostk0069) [inline ios/Sources/UIApplicationClient/Client.swift:10]: please add next line after

File: ios/Sources/AboutFeature/AboutDroidKaigiFeature.swift [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    case opensourceLicense
+    case privacyPolicy
+}
+
+public struct AboutDroidKaigiState: Equatable {
+}
+
+public enum AboutDroidKaigiAction {
+    case tap(AboutDroidKaigiModel)
+    case resultNavigation(Result<Bool, Never>)
```
Comment (ostk0069) [inline ios/Sources/AboutFeature/AboutDroidKaigiFeature.swift:15]: I think openApplicationSettings is better for action name

**Comment by ry-itto** [comment]
@ostk0069 
I fixed 👍🏼

**Review by ostk0069 — APPROVED** [review]

**Pull request merged**